src/backend/loader.js

```diff
@@ -31,6 +40,8 @@ if (sentryEnvironment !== 'production') {
   enableLogs = true
 }
 
+const sentryDB = new Database(dbDir)
```

Comment:
just to note that the parameter for `new Database()` is the path to a file, not a dir. The database will be created at the path `${dataDir}/sentry-logs`. I suggest naming it with `.db` extension to match what we've done with other databases.